docs/Config_Reference.md

```diff
@@ -1817,6 +1817,10 @@ z_offset:
 #   samples. If this tolerance is exceeded then either an error is
 #   reported or the attempt is restarted (see
 #   samples_tolerance_retries). The default is 0.100mm.
+#samples_discard_first: 0
+#   The number of samples to discard when starting to probe. This can
```

Comment:
To me this is a bit confusing as a description.

What a non-zero value would do is add N number of samples that are discarded. In other words, it's a bit confusion whether the value here is taken out of 'samples' or not.